src/todo.ts

```diff
@@ -96,4 +96,7 @@ class TodoApp {
         return this.toDoList.get(id);
     }
 }
+// Test comment for github purposes
+// Another test comment, this time after getting ssh figured out
+// A third test comment, for a third branch
```

Comment:
Looks good.  Let's do a little cleanup and do a tsdoc description of your class.  TSDoc looks like this:
```
/**
 * I am a description of a class
 */
```
Please do this on your class and remove the comments if you so choose.  Post the change here when done.

Reply:
Done.

Reply:
Excellent.